lib/network_config.ml

```diff
@@ -76,20 +76,25 @@ let read_management_conf () =
 				ports = [device, {default_port with interfaces = [device]}];
 				persistent_b = true
 				} in
-			match vlan with
-			| None ->
+			if bridge_name = "" then
 				[device, phy_interface; bridge_name, bridge_interface],
```

Comment:
Well, I don't think it would work, because you can have interfaces/bridges called "".